source/views/transportation/bus/map.js

```diff
@@ -68,13 +68,13 @@ export class BusMapView extends React.PureComponent {
     longitudeDelta: number,
   }) => {
     this.setState(state => {
-      const initialRegion = state.initialRegion
+      const newRegion = state.region
 
-      if (initialRegion && isEqual(initialRegion, region)) {
+      if (newRegion && isEqual(newRegion, region)) {
         return state
       }
 
-      return {initialRegion: region}
+      return {newRegion: region}
```

Comment:
Flip these, or else you'll be setting "this.state.newRegion". 

I think you want to call the input parameter to the onRegionChangeDomplete function "newRegion", and the one you get out of state "region".